plugins/scaffolder-node/src/tasks/types.ts

```diff
@@ -23,7 +24,11 @@ import { JsonObject, Observable } from '@backstage/types';
  * @public
  */
 export type TaskSecrets = Record<string, string> & {
+  /**
+   * @deprecated use "initiatorCredentials" instead
+   */
   backstageToken?: string;
+  initiatorCredentials?: string;
```

Comment:
I think it's best to keep this out of the secrets type, the way to interact with it is through `getInitiatorCredentials()`

Reply:
Removed